lib/perl/Genome/InstrumentData/AlignmentResult/Merged/External.pm

```diff
@@ -0,0 +1,133 @@
+package Genome::InstrumentData::AlignmentResult::Merged::External;
+
+use strict;
+use warnings;
+
+use Genome;
+
+use File::Basename qw();
+use File::Spec;
+
+
+class Genome::InstrumentData::AlignmentResult::Merged::External {
+    is => ['Genome::SoftwareResult::StageableSimple', 'Genome::InstrumentData::AlignedBamResult::Merged'],
+    has_input => [
+        sample => {
+            is => 'Genome::Sample',
+            doc => 'The sample from which the data for these alignments were taken',
+        },
+        file_content_hash => {
+            is => 'Text',
+        },
+        instrument_data => {
+            is => 'Genome::InstrumentData',
+            doc => 'The instrument data used for these alignments (if any)',
+            is_optional => 1,
+            is_many => 1,
+        },
+        annotation_build => {
+            is => 'Genome::Model::Build::ImportedAnnotation',
+            doc => 'The annotation build used for these alignments (if any)',
+            is_optional => 1,
+        },
+    ],
+    has_optional_metric => [
+        original_file_path => {
+            is => 'Text',
+        },
+        description => {
+            is => 'Text',
+            doc => 'brief information about the alignment data',
+        },
+        filetype => {
+            is => 'Text',
+            doc => 'the type of alignment (expressed as file extension)',
+        },
+    ],
+    has_constant => [
+        bam_path => {
```

Comment:
For anyone else reviewing, there was already discussion to keep using `bam_path` and later refactor to something like `alignment_path` and support both BAM and CRAM filetypes.